server/routes/api/graphql.ts

```diff
@@ -10,6 +10,8 @@ import { User, IUser, IFormItem } from "../../schema";
 import { Branches, Tags, AllTags } from "../../branch";
 import { schema as types } from "./api.graphql.types";
 
+import escapeStringRegexp = require("escape-string-regexp");
```

Comment:
This is the old, deprecated way of importing modules. Please use `import * as escapeStringRegexp from "escape-string-regexp;`

Reply:
I get an error when I do that with ts, something about it's type file not being a module